app/relisten/(tabs)/artists/[artistUuid]/songs.tsx

```diff
@@ -1,16 +1,116 @@
-import { useArtistVenues } from '@/relisten/realm/models/venue_repo';
+import { useArtistSongs } from '@/relisten/realm/models/song_repo';
+import Flex from '@/relisten/components/flex';
+import { RefreshContextProvider } from '@/relisten/components/refresh_context';
+import { SubtitleRow, SubtitleText } from '@/relisten/components/row_subtitle';
+import RowTitle from '@/relisten/components/row_title';
+import { DisappearingHeaderScreen } from '@/relisten/components/screens/disappearing_title_screen';
+import { SectionedListItem } from '@/relisten/components/sectioned_list_item';
 import { useLocalSearchParams } from 'expo-router';
-import { Text, View } from 'react-native';
+import {
+  FilterableList,
+  FilterableListProps,
+} from '@/relisten/components/filtering/filterable_list';
+import { RelistenText } from '@/relisten/components/relisten_text';
+import Plur from '@/relisten/components/plur';
+import { FilteringProvider, Filter, SortDirection } from '@/relisten/components/filtering/filters';
+import { useHeaderHeight } from '@react-navigation/elements';
+import { Song } from '@/relisten/realm/models/song';
 
 export default function Page() {
-  const { uuid } = useLocalSearchParams();
-
-  const results = useArtistVenues(String(uuid));
+  const { artistUuid } = useLocalSearchParams();
+  const results = useArtistSongs(String(artistUuid));
   const { data } = results;
+  const headerHeight = useHeaderHeight();
 
   return (
-    <View style={{ flex: 1, width: '100%' }}>
-      <Text className="text-white">Songs</Text>
-    </View>
+    <RefreshContextProvider networkBackedResults={results}>
+      <DisappearingHeaderScreen
+        headerHeight={headerHeight}
+        ScrollableComponent={SongList}
+        songs={Array.from(data.songs)}
+        filterPersistenceKey={['artists', artistUuid, 'songs'].join('/')}
+      />
+    </RefreshContextProvider>
   );
 }
+
+interface SongListItemProps {
+  song: Song;
+}
+
+const SongListItem = ({ song }: SongListItemProps) => {
+  return (
+    <SectionedListItem>
+      <Flex column>
+        <RowTitle>{song.name}</RowTitle>
+        <SubtitleRow>
+          <SubtitleText>
+            {'Played at '}
```

Comment:
nit but worth getting rid of the `{'string'}` here, just `Played at`

Reply:
The reason I added (and I think need) the string is because I need a space after the word `at`.

Reply:
in that case, you can use `&nbsp;`, but if you prefer the string, I'm ok with it

Reply:
Ah didn't think of that option. I personally prefer the string still, but I will keep this in mind in the future. And if you do have a strong preference, happy to update.